### DISCUSSION TITLE
How to resize icons

### DISCUSSION THREAD

**Discussion by dogooddesignco** [tailwindlabs/heroicons#675] [Help]
What is the appropriate way to resize an icon? I am simply pasting the SVG code into my markup but I'd like to use the icon at a slightly larger size. Thanks!

**Comment by rimzzlabs** [comment]
Hi, if you are using tailwindcss, you can simply set the width and height with _tailwind spacing utility_ like for example:

```html
<svg class="w-12 h-12"
.....
/>
```

**Reply by js-desmarais** [reply]
Does this work in a vanilla CSS project too? I can't seem to resize it. My icon svg is this:
`<svg
									xmlns="http://www.w3.org/2000/svg"
									fill="none"
									viewBox="0 0 24 24"
									stroke-width="1"
									stroke="currentColor"
									class="size-6"
								>
									<path
										stroke-linecap="round"
										stroke-linejoin="round"
										d="m2.25 12 8.954-8.955c.44-.439 1.152-.439 1.591 0L21.75 12M4.5 9.75v10.125c0 .621.504 1.125 1.125 1.125H9.75v-4.875c0-.621.504-1.125 1.125-1.125h2.25c.621 0 1.125.504 1.125 1.125V21h4.125c.621 0 1.125-.504 1.125-1.125V9.75M8.25 21h8.25"
									/>
								</svg>`

**Reply by lu-bian144** [reply]
im having thsame problem, have u benn able to figure it out?

**Reply by Daveawb** [reply]
Modify the class. In the example @js-desmarais posted. `class="size-6"` modify this to `class="size-4"` for example and you'll notice the difference.
